src/components/AdminPanelList/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Proptypes from "prop-types";
+
+export default function AdminPanelList({ data, onDelete, onDisplay }) {
+  let lists;
+  const { channelLists, roomLists, teamLists } = data;
+
+  if (channelLists) {
+    lists = channelLists;
+  }
+
+  if (roomLists) {
+    lists = roomLists;
+  }
+
+  if (teamLists) {
+    lists = teamLists;
+  }
+
+  return (
+    <ul>
+      {lists.map((channel) => {
+        const { _id: id } = channel;
+        return (
+          <li key={id}>
```

Comment:
```suggestion
          <li key=`channel_${id}`>
```
이처럼 key를 유일하게 식별가능하도록 하면 어떨지 제안드려요!

Reply:
저 key는 화면에 나타나는 부분이 아니라서 어떤 말씀인지 이해가 되진 않습니다 .

Reply:
_id가 eslint에 막혔었는데 저런 방법도 있군요 참고하겠습니다!